airbyte-webapp/package.json

```diff
@@ -50,6 +50,7 @@
     "lodash": "^4.17.21",
     "mdast": "^3.0.0",
     "query-string": "^6.13.1",
+    "rc-progress": "^3.4.1",
```

Comment:
The implementation for a progress bar is rather simple SVG. We should not use a library for that, but instead write a component in `components/ui` for this. Let me know if you want a FE dev to build that for you.